match/spec/storage/s3_storage_spec.rb

```diff
@@ -109,4 +109,36 @@
       end
     end
   end
+
+  describe Match::Storage::S3Storage do
```

Comment:
There is already a `describe Match::Storage::S3Storage do` at the beginning of this file 😄 we can remove this one (adjusting the indentation of course)

Reply:
I tried but it didn't work:

```
    #initialize with custom endpoint and force_path_style
      passes endpoint and force_path_style to S3ClientHelper (FAILED - 1)

Failures:

  1) Match Match::Storage::S3Storage#initialize with custom endpoint and force_path_style passes endpoint and force_path_style to S3ClientHelper
     Failure/Error: allow(subject).to receive(:s3_client).and_return(s3_client)
     
     NameError:
       undefined local variable or method `s3_client' for #<RSpec::ExampleGroups::Match::MatchStorageS3Storage::InitializeWithCustomEndpointAndForcePathStyle "passes endpoint and force_path_style to S3ClientHelper" (./match/spec/storage/s3_storage_spec.rb:129)>
     # ./match/spec/storage/s3_storage_spec.rb:8:in `block (3 levels) in <top (required)>'

Finished in 0.01108 seconds (files took 1.2 seconds to load)
9 examples, 1 failure

Failed examples:

rspec ./match/spec/storage/s3_storage_spec.rb:129 # Match Match::Storage::S3Storage#initialize with custom endpoint and force_path_style passes endpoint and force_path_style to S3ClientHelper
```

Reply:
I pushed this change and we'll see the error message from the CI